node-rfc/Dockerfile

```diff
@@ -21,17 +21,22 @@ COPY cts ${SAPBIN}/cts
 WORKDIR ${SAPBIN}
 
 RUN apt-get update && \
-    apt-get install -y apt-utils bzip2 unzip && \
-    rm -rf /var/lib/apt/lists/* /var/cache/apt/* && \
-    mkdir -p ${SAPDATADIR} && \
-    chown 1000:1000 ${SAPDATADIR} &&\
-    chmod a+x ${SAPBIN}/cts && \
-    cd ${SAPDIR} && \
-    unzip /tmp/$NWRFC_FILE && \
+    apt-get install -y apt-utils bzip2 unzip --no-install-recommends && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+WORKDIR  ${SAPDATADIR} 
+RUN     chown 1000:1000 ${SAPDATADIR} &&\
+    chmod a+x ${SAPBIN}/cts
+
+WORKDIR  ${SAPDIR} 
```

Comment:
Is the workdir statement here required? The two statement executed before the next WORKDIR are operating upon absolut directories. The workdir does not matter IMO.

Reply:
Yes it is. We need to be in that WORKDIR for the unzip und ldconfig command.

Reply:
not sure about `ldconfig` (how this relates to the current workdir), for unzip we have the `-d <DIR>`. In case `ldconfig` does not rely on the working dir we would be able to remove one layer.

Reply:
the ldconfig "adds" the shared libraries I think via symlinks into a central registry that the dynamik linker can load them. I didn't get it run in another via as this one.